src/parser/jobs/ast/modules/Draw.tsx

```diff
@@ -139,33 +159,24 @@ export class Draw extends Analyser {
 		this.checklist.add(new Rule({
 			displayOrder: DISPLAY_ORDER.DRAW_CHECKLIST,
 			name: <Trans id="ast.draw.checklist.name">
-				Play as many cards as possible
+				Play as many cards as possible considering <DataLink action="DIVINATION" /> windows.
```

Comment:
Probably want to revise the checklist name and description now that it's truly focusing on Draw usage. Otherwise lgtm.

Reply:
good catch! thank you
updated